README.md

```diff
@@ -2527,8 +2528,37 @@ With the default `TargetActor` classes, `Actors` are only valid targets when the
 
 **[⬆ Back to Top](#table-of-contents)**
 
+<a name="concepts-target-data-filters"></a>
+#### 4.11.3 Target Data Filters
+Using both the `Make GanmeplayTargetDataFilter` and `Make Filter Handle` nodes, you can filter out the players's `Pawn` or select only a specific class. If you need more advanced filtering, you can subclass `FGameplayTargetDataFilter` and override the `FilterPassesForActor` function. 
```

Comment:
`Make GanmeplayTargetDataFilter` <-- typo

Reply:
`players's` --> `player's`

Reply:
Good catch! Thanks :)

Reply:
sure thing, stranger.  Thanks for helping with the docs :)